test/artifact/inspec_compliance_test.rb

```diff
@@ -0,0 +1,12 @@
+require_relative "artifact_helper"
+
+class TestInspecCompliance < Minitest::Test
+  def test_compliance_version
+    command = "/bin/inspec compliance version #{TEST_CLI_OPTS}"
```

Comment:
Note that `version` is a live-fire API command - it tries to fetch the *server* API version. If you want an innocuous command, you might use `help` or `logout`.